assets/images/amp-black-icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Copied and modified from the 'Logos' link in the bottom right of https://amp.dev/